cfgov/regulations3k/models/django.py

```diff
@@ -16,6 +17,16 @@
 import regdown
 
 
+# Labels always require at least 1 alphanumeric character, then any number of
+# alphanumeric characters and hyphens.
+label_re_str = r'[\w]+[-\w]*'
+validate_label = RegexValidator(
+    re.compile(r'^' + label_re_str + r'$'),
+    'Enter a valid “label” consisting of letters, numbers, and hyphens.',
```

Comment:
Might help to add "with no spaces"
Same with sections.

Reply:
Done.

Reply:
Not seeing this change.

Reply:
Perhaps:

```suggestion
        'Enter a valid “label” consisting of letters, numbers, hyphens, and no spaces.',  # noqa
```

Reply:
Oh, sorry, I missed this one.